cmake/AddEventLibrary.cmake

```diff
@@ -11,10 +12,10 @@ macro(set_event_shared_lib_flags LIB_NAME)
 endmacro()
 
 macro(generate_pkgconfig LIB_NAME)
-    set(prefix      ${CMAKE_INSTALL_PREFIX})
-    set(exec_prefix ${CMAKE_INSTALL_PREFIX})
-    set(libdir      ${CMAKE_INSTALL_LIBDIR})
-    set(includedir  ${CMAKE_INSTALL_INCLUDEDIR})
+    set(prefix      "${CMAKE_INSTALL_PREFIX}")
+    set(exec_prefix "\${prefix}")
+    set(libdir      "\${prefix}/${CMAKE_INSTALL_LIBDIR}")
+    set(includedir  "\${prefix}/${CMAKE_INSTALL_INCLUDEDIR}")
```

Comment:
@ibauersachs what do you think, maybe here [`CMAKE_INSTALL_FULL_*`](https://cmake.org/cmake/help/v3.0/module/GNUInstallDirs.html) should be used instead?

Reply:
No, absolutely not. The whole point of this PR was to remove absolute paths.

Reply:
Ah, I see, here prefix is an escape, that will be interpreted by pkg-config.